demos/button.html

```diff
@@ -189,5 +189,26 @@
       </section>
     </main>
     <script src="assets/material-components-web.js" charset="utf-8"></script>
+    <script>
+      // Because we load our CSS via webpack, we need to ensure that all of the correct styles
```

Comment:
Agreed that I think we should reconsider our demo infrastructure to be more straightforward about how styles are loaded.